### PR TITLE
fix bug with creation time in ParseFilter()

### PR DESCRIPTION
**Pull request by 14rcole**
Previously, the creation time was set to the time the image was downloaded, but that was being compared to the time the image was built in matchesSinceImage() and matchesBeforeImage().  Updated the time to be the time that the image was built

### CODE REVIEW

**Comment by 14rcole** [comment]
@nalind PTAL

**Comment by nalind** [comment]
LGTM.

File: libkpod/image/image.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func ParseFilter(store storage.Store, filter string) (*FilterParams, error) {
 	}
 	params := new(FilterParams)
 	filterStrings := strings.Split(filter, ",")
+	fmt.Println(filterStrings)
```
Comment (runcom) [inline libkpod/image/image.go]: Leftover?

Comment (14rcole) [inline libkpod/image/image.go]: Yes, thank you.  Just pushed up the change

**Comment by rhatdan** [comment]
BTW, why didn't the tests run on Fedora?

**Comment by runcom** [comment]
should come back again with Fedora CI tests

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**